src/main/java/io/fabric8/maven/docker/StopMojo.java

```diff
@@ -82,40 +89,56 @@ private void stopContainers(QueryService queryService, RunService runService, Po
 
     // If naming strategy is alias stop a container with this name, otherwise get all containers with this image's name
     private List<Container> getContainersToStop(QueryService queryService, ImageConfiguration image) throws DockerAccessException {
-        List<Container> containers;
+        List<Container> containers = Collections.emptyList();
         RunImageConfiguration.NamingStrategy strategy = image.getRunConfiguration().getNamingStrategy();
-        if (strategy == RunImageConfiguration.NamingStrategy.alias) {
-            Container container = queryService.getContainer(image.getAlias());
-            if (container !=  null) {
-                containers = Collections.singletonList(container);
+        
+        //Do not fail if container is not found
+        try {
+            
+            if (strategy == RunImageConfiguration.NamingStrategy.alias) {
+                Container container = queryService.getContainer(image.getAlias());
+                if (container !=  null) {
+                    containers = Collections.singletonList(container);
+                }
             } else {
+                containers = queryService.getContainersForImage(image.getName());
+            }
+            
+            if(containers == null) {
                 containers = Collections.emptyList();
             }
-        } else {
-            containers = queryService.getContainersForImage(image.getName());
+        } catch (Exception ex) {
```

Comment:
Shouldn't this be more specific and e.g. around `queryService.getContainer()` ? (since the other method wont throw an exception in case of an unknown name)